src/Core/Graphics/LevelSprite.cpp

```diff
@@ -79,7 +79,8 @@ namespace obe::Graphics
         {
             m_path = path;
             const std::string fPath = System::Path(path).find();
-            m_texture = ResourceManager::GetTexture(fPath);
+            Debug::Log->debug("antiAliasing at loadTexture: {}", m_antiAliasing);
+            m_texture = ResourceManager::GetTexture(fPath, m_antiAliasing);
```

Comment:
Did you tested loading the same texture twice with antialiasing enabled and disabled ?